debian/README.Debian

```diff
@@ -0,0 +1,17 @@
+mumbledj
+========
+
+After installing the package, you must create
+/etc/systemd/system/mumbledj.service.d/local.conf containing the following:
+
+    [Service]
+    ExecStart=
+    ExecStart=/usr/sbin/mumbledj -server=mumble.example.com -password=secret "-channel=The Lounge" -username=mumbledj
+    #Environment=PATH=/var/lib/mumbledj/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+You must also configure your Youtube and/or Soundcloud API keys by editing
```

Comment:
Minor nitpicks:
* "Youtube" should be "YouTube"
* "Soundcloud" should be "SoundCloud"